backend/node/genvm/base.py

```diff
@@ -302,11 +299,11 @@ async def consume_result(
     ) -> None:
         if type == ResultCode.RETURN:
             self._result = ExecutionReturn(ret=bytes(data))
-        elif type == ResultCode.ROLLBACK:
```

Comment:
I decided to rename this result codes to prevent future confusion. Name `USER_ERROR` stands for error that contract produced explicitly. But name `contract_error` would bring a lot of confusion right now due to a clash with previous naming